### ISSUE TITLE
After installation, game will not run

### ISSUE THREAD

**Issue by BucketListMod** [Nexus-Mods/NexusMods.App#1759]
# Bug Report

## Summary
After adding and applying mods, "Play" button is grey and does not function. Attempting to play from Steam causes button to go "Play", "Cancel", "Stop", and back to "Play" without opening SMAPI or Stardew

## Steps to reproduce
1. Steamdeck set up to play with manual SMAPI installation
2. Uninstall SMAPI and all mods
3. Log in and add mods via .nxm links to Nexus app
4. Apply
5. Play no longer functions

## What is the expected behaviour?
To be allowed to play :c

## Other information

[nexusmods.app.main.current.log](https://github.com/user-attachments/files/16229082/nexusmods.app.main.current.log)
[nexusmods.app.slim.current.log](https://github.com/user-attachments/files/16229083/nexusmods.app.slim.current.log)

**Comment by BucketListMod** [comment]
Spotfix: Manually uninstalled and reinstalled SMAPI. Still strange it didn't work when installed through the app

**Comment by LukeNexusMods** [comment]
I just wanted to check for this did you start with a clean installation with no mods? 

At the moment our recommendation is that you start completely clean and install all mods and SMAPI through the app.

**Comment by BucketListMod** [comment]
Yep, I always delete my Mods folder (after zipping and moving elsewhere) and uninstall SMAPI. Also had a general problem with the mods not deploying correctly, suspecting volume may be an issue (about to open a new issue about that specifically)

**Comment by LukeNexusMods** [comment]
I just wanted to check to see if this was still an issue?
